packages/core/test/convert.spec.ts

```diff
@@ -88,9 +89,7 @@ describe('Convert EthersJS Objects', () => {
         throw new Error(`Unsupported chain ID: ${chainId}`)
       }
 
-      try {
-        fetchURLForNetwork(BigInt(chainId))
-      } catch {
+      if (!isLiveNetworkRpcApiKeyDefined(BigInt(chainId))) {
```

Comment:
These lines serve the same purpose except `isLiveNetworkRpcApiKeyDefined` doesn't indiscriminately catch errors, which was an issue caused by the `try/catch`